### ISSUE TITLE
Missing 'Execute on Client' Option in Script block 

### ISSUE THREAD

**Issue by raylancordeiro** [baptisteArno/typebot.io#1446]
Hello,

I've been trying to use the setVariable() function in the scriptblock of Typebot, but I am encountering issues. The documentation suggests unchecking the "Execute on client" option to resolve this. However, this option no longer appears in the script block window interface. Could you please help me understand if this feature has been removed or relocated in the latest updates? Any guidance on how I can access this option or use the setVariable() function effectively would be greatly appreciated.

Thank you!

**Comment by raylancordeiro** [comment]
Hello,

I've noticed that there have indeed been changes to the script block and the setVariable component when comparing the two versions. I was able to make the necessary adjustments and everything is now working as intended. However, I would suggest an update to the documentation, as it still advises users to uncheck the "Execute on client" option in the script block, which no longer appears. An update would help clarify this for others.

https://docs.typebot.io/editor/blocks/logic/script#setvariable-function

Thank you for your attention to this matter!
